### DISCUSSION TITLE
It was working very well, but since today this error has started to appear: ERROR - [send_minute_watched_events]: Exception raised in send minute watched Traceback (most recent call last):  And a lot of text related to the sessions and models files

### DISCUSSION THREAD

**Discussion by Frehyyy** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#256] [Q&A] [answered]
It was working very well, but since today this error has started to appear:
ERROR - [send_minute_watched_events]: Exception raised in send minute watched
Traceback (most recent call last):

And a lot of text related to the sessions and models files

And sometimes I get messages like the one on the screen
![image](https://user-images.githubusercontent.com/89162555/129965095-a9af2676-734e-4536-bb0b-e776ac92c4b7.png)

plz help

**Comment by Tkd-Alex** [comment, marked as answer]
It's a problem related to your internet connection.

**Reply by Frehyyy** [reply]
I have a good Internet, 100 mbps, I tried restarting the Internet, computer, now works properly, but sometimes still comes out the error that was on the screen. It does not prevent work, but there is

**Comment by 1v** [comment]
I had same error at same day but only one.
